pkg/accounting/tally/tally.go

```diff
@@ -98,16 +98,20 @@ func (t *tally) identifyActiveNodes(ctx context.Context) (err error) {
 				if err != nil {
 					return Error.Wrap(err)
 				}
-				pieces := pointer.Remote.RemotePieces
-				var nodeIDs storj.NodeIDList
-				for _, p := range pieces {
-					nodeIDs = append(nodeIDs, p.NodeId)
+				if pointer.Remote == nil {
```

Comment:
nit --
```
if pointer.Remote == nil {
    t.logger.Warn("MISSING pointer.Remote")
    return nil
}

pieces := pointer.Remote.RemotePieces
...

Reply:
This shouldn't warn on inline segments. We are expected to have tons of them.

Reply:
I'm covering this in my pr...https://github.com/storj/storj/pull/820